tools/ClassGen/NodeBuilder.h

```diff
@@ -45,6 +110,8 @@ class NodeBuilder {
   std::vector<std::pair<std::string, std::string>> nodeOutputs_;
   /// A list of node members. Format: (type, name).
   std::vector<std::pair<MemberType, std::string>> members_;
+  /// A list of external enum node members. Format: (EnumBuilder*, name).
+  std::vector<std::pair<const EnumBuilder *, std::string>> enumMembers_;
```

Comment:
You add enum members to this separate list, different from `members_`. Therefore, the order of members in the generated class will be different from the order of members as it is provided in the NodeGen. May be such members should be added to the `members_`? E.g. `MemberType` would become something more complex than a simple enum describing a predefined set of types and  `EnumBuilder` should inherit from `MemberType`.